test/helpers.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+const BITCOIN_BLOCK_HEADER_SIZE = require('../src/index')
+  .util.BITCOIN_BLOCK_HEADER_SIZE
+
+const headerFromHexBlock = (hex) => {
+  return Buffer.from(hex.toString(), 'hex').slice(0, BITCOIN_BLOCK_HEADER_SIZE)
```

Comment:
Seems inefficient in time and memory. What is the `hex` type?  Why the ToString()?

If `hex` is a string, then it would be better to first take the BITCOIN_BLOCK_HEADER_SIZE * 2 characters and then create the Buffer from it.

If this method is just used by tests, then you can ignore the above.

Reply:
I'ts just for tests. The Bitcoin blocks are serialized as "hex", it's bytes encoded as hex put into a file. That's what the Go implementation is using and i didn't want to convert the files, but just use the same ones for the JS implementation as well.